src/api/ingest/fishbelt_serializer.py

```diff
@@ -81,7 +81,8 @@ class FishBeltCSVSerializer(CollectRecordCSVSerializer):
         choices=tide_choices, required=False, allow_null=True, allow_blank=True
     )
 
-    data__fishbelt_transect__len_surveyed = serializers.IntegerField(min_value=0)
+    # data__fishbelt_transect__len_surveyed = serializers.IntegerField(min_value=0)
+    data__fishbelt_transect__len_surveyed = serializers.DecimalField(max_digits=4, decimal_places=1)
```

Comment:
Looks like you've done fishbelt and benthic PIT; but we should also do benthic LIT and habitatcomplexity